### PR TITLE
[5.6] Add groupBy higher order proxy to changelog

### PR DESCRIPTION
**Pull request by amaelftah**

### CODE REVIEW

File: CHANGELOG-5.6.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 ### Added
 - Added `view:cache` command ([9fd1273](https://github.com/laravel/framework/commit/9fd1273ad79a46bb3aa006129109c6bc72766e4b), [2ab8acf](https://github.com/laravel/framework/commit/2ab8acfef5d7e784148b2367b5bcf083a0d0d024))
-- Added `min()` and `max()` to as higher order proxies ([#23560](https://github.com/laravel/framework/pull/23560))
+- Added `min()` and `max()` and `groupBy()` to as higher order proxies ([#23560](https://github.com/laravel/framework/pull/23560), [#23608](https://github.com/laravel/framework/pull/23608))
```
Comment (jmarcher) [inline CHANGELOG-5.6.md]: Added `min()`, `max()` and `groupBy()` to Collections as higher order proxies

Comment (Te7a-Houdini) [reply CHANGELOG-5.6.md]: @jmarcher Thanks

File: CHANGELOG-5.6.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 ### Added
 - Added `view:cache` command ([9fd1273](https://github.com/laravel/framework/commit/9fd1273ad79a46bb3aa006129109c6bc72766e4b), [2ab8acf](https://github.com/laravel/framework/commit/2ab8acfef5d7e784148b2367b5bcf083a0d0d024))
-- Added `min()` and `max()` to as higher order proxies ([#23560](https://github.com/laravel/framework/pull/23560))
+- Added `min()` and `max()` and `groupBy()` to as Collection higher order proxies ([#23560](https://github.com/laravel/framework/pull/23560), [#23608](https://github.com/laravel/framework/pull/23608))
```
Comment (jmarcher) [inline CHANGELOG-5.6.md]: Now is even worse :man_facepalming:

File: CHANGELOG-5.6.md [hunk 3]
```diff
@@ -4,7 +4,7 @@
 
 ### Added
 - Added `view:cache` command ([9fd1273](https://github.com/laravel/framework/commit/9fd1273ad79a46bb3aa006129109c6bc72766e4b), [2ab8acf](https://github.com/laravel/framework/commit/2ab8acfef5d7e784148b2367b5bcf083a0d0d024))
-- Added `min()` and `max()` to as higher order proxies ([#23560](https://github.com/laravel/framework/pull/23560))
+- Added `min()` and `max()` and `groupBy()` to Collection as higher order proxies ([#23560](https://github.com/laravel/framework/pull/23560), [#23608](https://github.com/laravel/framework/pull/23608))
```
Comment (jmarcher) [inline CHANGELOG-5.6.md]: Change first and for a comma.

**Pull request closed without merging**